src/main/java/com/kainos/ea/enums/Bands.java

```diff
@@ -0,0 +1,25 @@
+package com.kainos.ea.enums;
+
+public enum Bands {
+    BAND1("BAND1"), BAND2("BAND2"), BAND3("BAND3");
```

Comment:
Nit: New line for each ENUM element is better.

Reply:
I would also advise 'BAND_1' over 'BAND1'

Reply:
done